app/steps/ui/shutterpage/template.html

```diff
@@ -0,0 +1,30 @@
+{% extends "includes/layout.html" %}
+{% block page_title %}
+{{ content.title }}
+{% endblock %}
+
+{% set displaySignOutLink = 'false' %}
+{% set serviceName = '' %}
+
+{% block content %}
+
+<main id="content" role="main">
+    {% include "includes/phase_banner_beta.html" %}
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <h1 class="form-title heading-large">{{content.header}}</h1>
+            <p>{{content.introduction | safe}}</p>
+            <p>{{content.paragraph1 | safe}}</p>
+            <p>{{content.paragraph2}}</p>
+            <p>{{content.paragraph3 | safe}}</p>
+            <h2 class="heading-medium">
+                {{content.contactHeader}}
+            </h2>
+            <p>{{content.paragraph4 | safe}}</p>
+            <p>{{content.paragraph5 | safe}}</p>
+        </div>
+    </div>
+
+</main>
+
+{% endblock %}
```

Comment:
Can we have a component test for this page?

It needs to check the content on the page is correct.